.circleci/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:xenial
+
+RUN \
+  apt-get update &&\
+  apt-get install -y locales &&\
+  locale-gen en_US.UTF-8 &&\
+  update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+
+RUN \
+  apt-get install -y openjdk-8-jdk
+
+RUN \
+  echo "deb http://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list &&\
+  apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823 &&\
+  apt-get update &&\
+  apt-get install -y sbt
```

Comment:
I manually build the image from the Dockerfile, everything went OK.
A CircleCI v2 build [also succeeded](https://circleci.com/gh/input-output-hk/mantis/3495).